src/component/ProductsFilter/ProductsFilter.jsx

```diff
@@ -0,0 +1,116 @@
+import "./productFilter.css";
+
+const ProductsFilter = () => {
+  return (
+    <>
+      <div className='sidebar-header'>
+        <h2>Filters</h2>
+        <p className='md-margin'>Clear</p>
+      </div>
+      <h3>Price</h3>
+      <div className='filter'>
+        <label htmlFor='radio-1'>
+          <input
+            className='margin'
+            id='radio-1'
+            name='radio'
+            type='radio'
+            onClick={() =>
+              dispatch({ type: "SORTING", payload: "low to high" })
+            }
+          />
+          Low to High
+        </label>
+        <label htmlFor='radio-2'>
+          <input
+            className='margin'
+            id='radio-2'
+            name='radio'
+            type='radio'
+            onClick={() =>
+              dispatch({ type: "SORTING", payload: "high to low" })
+            }
+          />
+          High to Low
+        </label>
+      </div>
+      <h3>Category</h3>
+      <div className='filter'>
+        <label htmlFor='checkbox-1'>
+          <input
+            className='margin'
+            id='checkbox-1'
+            name='checkbox'
+            type='checkbox'
+            onClick={() => dispatch({ type: "GRAINS" })}
+          />
```

Comment:
Here you can give meaningful values to id and name. For e.g, if the checkbox is for grains you can give id="grains" and name="grains"